// 118
fn main() -> Int {
    let x: Byte!Int = .err('d')
    x catch 118
}
